package.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "@azure/ng-deploy",
   "version": "0.1.0-beta.3",
+  "main": "out/index.js",
```

Comment:
I like having `index.js` as the entry point. However, after i run `npm run build` no file exists in the `/out` folder (just sub folders). I think we should have an entry point ... right?